src/app/static/js/warning.js

```diff
@@ -0,0 +1,31 @@
+let MAX_FILE_SIZE = 10485760; // 10 MB in bytes
+let MAX_FILE_SIZE_STR = "10MB";
+function checkFileSize(file, warningElementJQ) {
```

Comment:
Looking at issue #306 and the attached file, 6MB will cause issues.  Let's go with 5MB as the size for a warning.

Reply:
Fixed it 👍